tools/exportNotebook

```diff
@@ -93,33 +283,96 @@ then
+        <script type="application/ecmascript"> <![CDATA[
+            var visiblePage = 'p1';
+            function goToPage(page) {
+                document.getElementById(visiblePage).setAttribute('style', 'display: none');
+                document.getElementById(page).setAttribute('style', 'display: inline');
+                visiblePage = page;
+            }
+        ]]> </script>
+    <g id="p1" style="display:inline">
+<rect x="0" y="0" width="1404" height="1872" fill-opacity="0"/></g></svg>
+EOF
+    fi
+done
+
+# Preprocess foreground*.svg
+for f in "${tmpfolder}"/foreground*.svg; do
+    # debug
+    #cp -v "$f" ~/debug/"$(basename "$f")" 
+    sed -i "s/stroke-width\:[\.0-9]\+/stroke-width:${STROKE_WIDTH}/g;s/opacity\:[\.0-9]\+/opacity:${OPACITY}/g" "$f"
+    [ ! $? -eq 0 ] && echo "Failed to reset stroke-width and opacity in '$f'" && exit 1
+done
+# - optimize svg with svgo
+if command -v "svgo" > /dev/null; then
+    for f in "${tmpfolder}"/foreground*.svg; do
+        svgo -i "$f"
+        [ ! $? -eq 0 ] && echo "Failed to optimize svg '$f'" && exit 1
+    done
+fi
 
-if command -v "rsvg-convert" > /dev/null
-then
+# Convert svg to pdf
+if command -v "inkscape" > /dev/null; then
+    for f in "${tmpfolder}"/foreground*.svg; do
+        inkscape "$f" -d 200 --export-pdf="${f%.svg}.pdf"
+        [ ! $? -eq 0 ] && echo "Failed to convert to pdf via inkscape '$f'" && exit 1
+    done
+    pdftk "${tmpfolder}"/foreground*.pdf cat output "${tmpfolder}"/foreground.pdf
+    [ ! $? -eq 0 ] && echo "Failed to merge inkscape pdfs" && exit 1
+elif command -v "rsvg-convert" > /dev/null; then
     rsvg-convert -a -f pdf "${tmpfolder}"/foreground*.svg -o "${tmpfolder}"/foreground.pdf
+    [ ! $? -eq 0 ] && echo "Failed to convert to pdf via rsvg-convert" && exit 1
 else
     convert -density 100 "${tmpfolder}"/foreground*.svg -transparent white "${tmpfolder}"/foreground.pdf
+    [ ! $? -eq 0 ] && echo "Failed to convert to pdf via convert" && exit 1
 fi
 
-# Strip .pdf suffix if it already exists (document vs. notebook)
-filename=$(basename -s .pdf ${filename//\"/})
-
 # Use multistamp instead of multibackground to preserve transparency
 pdftk "${tmpfolder}"/background.pdf multistamp "${tmpfolder}"/foreground.pdf output "${filename}.pdf"
+[ ! $? -eq 0 ] && echo "Failed to merge background and foreground pdfs" && exit 1
 
 filesize=$(ls -la "${filename}.pdf" | awk '{print $5}' | numfmt --to=iec-i --suffix=B --format="%.3f")
+
 echo "Written ${filesize} to ${filename}.pdf"
 
-ssh $control_options -O exit ${SSH_IP}
-rm -Rf "${tmpfolder}" "${control_path_dir}"
+if [ -z $KEEP_TEMPDIR ]; then
+    rm -Rf "${tmpfolder}"
+else
+    echo "Left temporary dir '${tmpfolder}'"
+fi
+
+if [ ! -z $ASK_RENAME ]; then
+    xdg-open "${filename}.pdf"
+    [ ! -z "$OUTPUT_UUID" ] && echo "Last modified: ${DOCUMENTS_DATES[$OUTPUT_UUID]}"
+    echo
+    read -rp "New name w/o extension, or press Enter to delete files: " NEWFILENAME
```

Comment:
The "Enter to delete" action is surprising. When asked for an entry, empty Enter usually accepts the default, or keeps the current value unchanged.

Reply:
Good point! I was carried away by optimizing my workflow to the detriment of wider use. Now enter will continue and "!rm" will delete the files.